src/resources/SenderResource.ts

```diff
@@ -0,0 +1,144 @@
+    const request = this.api.get('/api/account/senders');
+
+    if (params) {
+      request.withQueryParams({ ...params });
+    }
+
+    const response = await request.fetch();
+
+    return (await response.json()) as Responses.BulkGetResponse<Sender>;
+  }
+
+  /**
+   * Retrieves detailed Account information about a specified user.
+   *
+   * @param id - Unique sender ID
+   *
+   * @remarks
+   * - {@link https://community.onespan.com/products/onespan-sign/sandbox#/Senders/api.account.senders._senderId.get | REST API documentation (OneSpan)}
+   *
+   * - {@link https://community.onespan.com/documentation/onespan-sign/guides/feature-guides/developer/managing-senders | Managing Senders (OneSpan)}
+   */
+  public async getOne(id: string): Promise<Sender> {
+    const response = await this.api.get(`/api/account/senders/${id}`).fetch();
+    return (await response.json()) as Sender;
+  }
+
+  /**
+   * Deletes a specified Sender from an account.
+   *
+   * @param id - Unique sender ID
+   *
+   * @remarks
+   * - {@link https://community.onespan.com/products/onespan-sign/sandbox#/Senders/api.account.senders._senderId.delete | REST API documentation (OneSpan)}
+   *
+   * - {@link https://community.onespan.com/documentation/onespan-sign/guides/feature-guides/developer/managing-senders | Managing Senders (OneSpan)}
+   */
+  public async delete(id: string): Promise<void> {
+    await this.api.delete(`/api/account/senders/${id}`).fetch();
+  }
+
+  /**
+   * Sends an email invitation to a user. The user will receive an invitation link and a token identifying the account.
+   *
+   * @param senderId - Unique sender ID
+   *
+   * @remarks
+   * - {@link https://community.onespan.com/products/onespan-sign/sandbox#/Senders/api.account.senders._senderId.invite.post | REST API documentation (OneSpan)}
+   *
+   * - {@link https://community.onespan.com/documentation/onespan-sign/guides/feature-guides/developer/managing-senders | Managing Senders (OneSpan)}
+   */
+  public async invite(senderId: string): Promise<Sender> {
+    const response = await this.api.post(`/api/account/senders/${senderId}/invite`).fetch();
+    return (await response.json()) as Sender;
+  }
+
+  /**
+   * Retrieves the image that is being used by a recipient as their default signature when signing a transaction.
+   *
+   * @param senderId - Unique sender ID
+   *
+   * @remarks
+   * - {@link https://community.onespan.com/products/onespan-sign/sandbox#/Senders/api.account.senders._senderId.signature.image.get | REST API documentation (OneSpan)}
+   *
+   * - {@link https://community.onespan.com/documentation/onespan-sign/guides/feature-guides/developer/extracting-images-signatures | Extracting Images of Signatures (OneSpan)}
+   */
+  public async getDefaultSignature(senderId: string): Promise<Responses.SignatureImage> {
+    const response = await (await this.api.get(`/api/account/senders/${senderId}/signature/image`).fetch()).json();
```

Comment:
```suggestion
    const response = await this.api.get(`/api/account/senders/${senderId}/signature/image`).fetch()
    const signatureImage = await response.json()
```

nit – I'd prefer breaking up that expression :P